application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -33,19 +35,22 @@ public TagCommand(TagSystem tagSystem) {
 
         // TODO Thing about adding an ephemeral selection menu with pagination support
```

Comment:
Could you quickly edit the typo here? Its supposed to be `Think`, not `Thing`, thanks.